app/views/timeline_events/show.html.erb

```diff
@@ -2,55 +2,105 @@
 <% content_for(:head) do %>
   <title><%= presenter.page_title %></title>
 <% end %>
-<div class="bg-gray-50 px-3 py-4">
-  <div class="mb-2 container max-w-3xl mx-auto">
-    <div class="bg-white shadow p-4 rounded">
-      <div class="bg-white mb-2 p-3 flex flex-col">
-        <h2 class="font-semibold text-lg"> <%= @submission.title %></h2>
-        <p class="text-gray-600 text-xs" title="<%= @submission.created_at.iso8601 %>"> <%= t('.submitted_on', date: @submission.created_at.strftime('%b %d, %Y'))%></p>
-        <% if @submission.evaluated_at.present? %>
-          <% translation_key = @submission.passed_at.present? ? 'accepted_on' : 'rejected_on' %>
-          <p class="text-gray-600 text-xs mt-1" title="<%= @submission.evaluated_at.iso8601 %>"> <%= t(".#{translation_key}", date: @submission.created_at.strftime('%b %d, %Y'))%></p>
+<div class="bg-gray-50 pt-6">
+  <div class="max-w-2xl mx-auto px-5">
+    <div class="flex items-center gap-3">
+      <div class="bg-blue-100 rounded-full text-blue-500 h-10 w-10 flex items-center justify-center">
+        <i class="if i-journal-text-light if-fw text-xl"></i>
+      </div>
+      <p class="text-md font-semibold"><%= @submission.title %></p>
+    </div>
+    <div class="flex flex-wrap gap-5 py-5 divide-x-2">
+      <div class="pr-4">
+        <p class="text-xs text-gray-500"><%= t(".submission_from") %></p>
+        <p class="text-sm font-semibold"><%= @submission.founder.name %></p>
+      </div>
+      <div class="px-4">
+        <p class="text-xs text-gray-500"><%= t(".submitted_on") %></p>
+        <p class="text-sm font-semibold"> <%= @submission.created_at.strftime("%b %d, %Y")%></p>
+      </div>
+      <div class="px-4">
+        <p class="text-xs text-gray-500"><%= t(".reviewed_on") %></p>
+        <p class="text-sm font-semibold"><%= @submission.evaluated_at.present? ? @submission.evaluated_at.strftime("%b %d, %Y") : t(".not_available") %></p>
+      </div>
+      <div class="px-4">
+        <p class="text-xs text-gray-500"><%= t(".status") %></p>
+        <% if @submission.passed_at.present? %>
+          <p class="text-sm font-semibold text-green-500"><%= t(".accepted") %></p>
+        <% elsif  @submission.evaluated_at.present? %>
+          <p class="text-sm font-semibold text-red-500"><%= t(".rejected") %></p>
+        <% else %>
+          <p class="text-sm font-semibold text-blue-500"><%= t(".pending") %></p>
         <% end %>
       </div>
-      <% @submission.checklist.each_with_index do |item, index| %>
-        <div class="mb-3 bg-gray-50 rounded">
-          <div class="text-lg font-bold ps-3 pt-3"><%= t('.question_with_number', number: index + 1) %></div>
```

Comment:
Why did we remove the number?

Reply:
Suhail told me to use same design as used in review interface, seen by a student.